tests/tests/test_integration/test_detection.py

```diff
@@ -50,3 +61,36 @@ def test_detection_full():
     assert isclose(
         num_cells_validation, num_cells_test, abs_tol=DETECTION_TOLERANCE
     )
+
+
+def test_callbacks(signal_array, background_array):
+    # 20 is minimum number of planes needed to find > 0 cells
+    signal_array = signal_array[0:20]
+    background_array = background_array[0:20]
```

Comment:
why not do this in the `pytest.fixture` above?

Reply:
Because the fixture needs to return the full array for the pre-exsiting test above the one I added.

Reply:
Ah, so presumably they weren't using a fixture before?